### ISSUE TITLE
form items not work in dashboard

### ISSUE THREAD

**Issue by aghict** [almasaeed2010/AdminLTE#224]
Hello,
While we have demo.js and app.js any tag of <form> not work.
I check http://almsaeedstudio.com/AdminLTE/ and we have this problem too.

**Comment by almasaeed2010** [comment]
This issue has to do with the WYSIWYG plugin in the dashboard. There is a solution but it will be implemented with AdminLTE 2.0 which addresses many other major issues too. Thanks for reminding me!

**Comment by aghict** [comment]
Thanks.
Can you give me solution? I don't use WYSIWYG editor.

**Comment by kressly** [comment]
Hello @aghict
What he is trying to say is that you should download the latest version of AdminLte and that problem will be solved.
Kindly go to this adress

http://almsaeedstudio.com/themes/AdminLTE/AdminLTE2-dist.zip

Click on it to download the latest version that may solve ur problem

**Comment by almasaeed2010** [comment]
This issue has not been solved in 2.0 yet. The only realse of AdminLTE 2.0 is in its alpha and should not be used in production. I will inform you when this issue is solved and its solution is available.

**Comment by lenamtl** [comment]
You can try to remove the editor js script link and code from the page

**Comment by aghict** [comment]
I make RTL theme in version 1.0 and now I don't what to change my theme.

**Comment by almasaeed2010** [comment]
Fixed in 2.0.

Solution:
in dashboard.js:
remove ```.disableSelection()``` from .sortable elements (lines 23 and 31).
Final code from line 11 to 31:
```js
  //Activate the iCheck Plugin
  $('input[type="checkbox"]').iCheck({
    checkboxClass: 'icheckbox_flat-blue',
    radioClass: 'iradio_flat-blue'
  });
  //Make the dashboard widgets sortable Using jquery UI
  $(".connectedSortable").sortable({
    placeholder: "sort-highlight",
    connectWith: ".connectedSortable",
    handle: ".box-header, .nav-tabs",
    forcePlaceholderSize: true,
    zIndex: 999999
  });
  $(".connectedSortable .box-header, .connectedSortable .nav-tabs-custom").css("cursor", "move");
  //jQuery UI sortable for the todo list
  $(".todo-list").sortable({
    placeholder: "sort-highlight",
    handle: ".handle",
    forcePlaceholderSize: true,
    zIndex: 999999
  });
```

We appreciate your patience.
